server/routes/api/reviews.js

```diff
@@ -69,4 +69,39 @@ router.post("/requests", authenticate, async (req, res) => {
   res.sendStatus(500);
 });
 
+router.get("/requests", authenticate, async (req, res) => {
+  try {
+    const userId = req.user.id;
+
+    const usersRequests = await Review.find({ requesterId: userId });
+
+    const filteredRequests = usersRequests.map((request) =>
+      request.filteredSchema()
+    );
+
+    res.send({ usersRequests: filteredRequests });
+  } catch (err) {
+    console.error(err);
+  }
+});
+
+router.get("/requests/:id", authenticate, async (req, res) => {
+  const requestId = req.params.id;
+
+  try {
+    const userId = req.user.id;
+
+    const singleRequest = await Review.find({
+      _id: { $in: [requestId] },
+      requesterId: { $in: [userId] },
+    });
+
+    singleRequest[0].filteredSchema();
```

Comment:
you assuming it exists here, you should definitely check, and if it doesn't return a 404